metaquot/ppxlib_metaquot.ml

```diff
@@ -39,6 +39,30 @@ module Make(M : sig
       match t.ptyp_desc with
       | Ptyp_extension ({ txt = "t"; _}, _ as ext)-> M.cast ext
       | _ -> super#core_type t
+
```

Comment:
This change should probably not be part of this PR.

Reply:
(sorry, I meant the support for the new quotations)

Reply:
removed